infra-parent/infra-v2/src/main/java/com/redhat/service/smartevents/infra/v2/api/models/dto/BaseResourceDTO.java

```diff
@@ -0,0 +1,117 @@
+package com.redhat.service.smartevents.infra.v2.api.models.dto;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.redhat.service.smartevents.infra.v2.api.models.OperationType;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public abstract class BaseResourceDTO {
+    @JsonProperty("id")
+    private String id;
+
+    @JsonProperty("name")
+    private String name;
+
+    @JsonProperty("customerId")
+    private String customerId;
+
+    @JsonProperty("owner")
+    private String owner;
+
+    @JsonProperty("operation")
+    private OperationType operationType;
+
+    @JsonProperty("generation")
+    private long generation;
+
+    @JsonProperty("timeout")
+    private int timeout;
```

Comment:
nitpick: better to add the unit like `seconds` in the name to make the contract more self-descriptive

Reply:
@r00ta @robpblake renamed.